### ISSUE TITLE
Unable to de-select multi items using a screenreader/keys

### ISSUE THREAD

**Issue by sawyerclick** [rob-balfre/svelte-select#592]
**The problem**
While using a `multiple` dropdown, users are unable to de-select items using solely keyboard strokes. Users are able to do so via click events. This makes the dropdown inaccessible to screenreaders and keyboard-based users

**The fix**
In order to be accessible and match click events, the 'Enter' keyboard stroke that selects should toggle the selection state rather than solely selecting.

I'm willing to write a PR to fix, if required

**Comment by sawyerclick** [comment]
One obvious fix is to set keydown's `Enter` case to use `handleItemClick({item: hoverItem, i: hoverItemIndex})`

**Comment by rob-balfre** [comment]
You can use keybinding: left and right arrows then backspace to remove.

**Comment by rob-balfre** [comment]
Reopen if you think it needs updating still. Thanks
